brunch-config.js

```diff
@@ -52,6 +52,10 @@ exports.config = {
     babel: {
       // Do not use ES6 compiler in vendor code
       ignore: [/web\/static\/vendor/]
+    },
+    cleancss: {
+      keepSpecialComments: false,
+      removeEmpty: true
```

Comment:
I'm working on configuring the deployment right now https://github.com/edenspiekermann/espi-dni/pull/10 - we'll find out soon :)